plutus-scb/src/Plutus/SCB/Query.hs

```diff
@@ -128,3 +162,208 @@ utxoIndexProjection =
 
 blockCount :: forall t key position. Projection (Sum Integer) (StreamEvent key position (ChainEvent t))
 blockCount = contramap (const 1) monoidProjection
+
```

Comment:
A lot of this feels boilerplate-y. I don't have a concrete suggestion, though.

Reply:
I [re-wrote](https://github.com/input-output-hk/plutus/pull/1992/commits/6fb63e4c30d76809c02bbc23ceb69084093cb80d#diff-ea4da314db139ab07a28ce96abf5bf9fR274) the contract instance queries to use prisms, now they're all one-liners, and they're only used once, so I could inline them?

Reply:
Cool. I think having the names is still useful.